File: serverscript/customization/item_sum.js
frappe.ui.form.on('Item', {
    quantity: function(frm) {
      calculate_total_cost(frm);
    },
    unit_cost: function(frm) {
      calculate_total_cost(frm);
    }
  });
  
  function calculate_total_cost(frm) {
    if (frm.doc.quantity && frm.doc.unit_cost) {
      frm.set_value('total_cost', frm.doc.quantity * frm.doc.unit_cost);
    }
  }
  

File: serverscript/customization/item_sum.py
import frappe
from frappe.model.document import Document
def calculate_total_cost(doc, method):
    frappe.logger().info(f"Calculating total amount of total ")
    print("Hello JI ")
    for item in doc.custom_item_child_:
        if item.quantity and item.unit_cost:
            item.total_cost = item.quantity * item.unit_cost